lib/ops/opsservice/update.go

```diff
@@ -254,6 +254,14 @@ func (o *Operator) RotatePlanetConfig(req ops.RotatePlanetConfigRequest) (*ops.R
 	checks.OverrideDockerConfig(&dockerConfig,
 		checks.DockerConfigFromSchema(req.Manifest.SystemOptions.DockerConfig()))
 
+	env := req.Env
+	if env == nil {
+		env, err = o.getClusterEnvironment()
```

Comment:
nit: I would leave a comment that this maintains the existing configuration between planet configuration updates:
```suggestion
		// Keep existing environment configuration inside the container
		env, err = o.getClusterEnvironment()
```
same for cluster configuration below.